crates/red_knot_python_semantic/resources/mdtest/exception/control_flow.md

```diff
@@ -241,30 +241,34 @@ suites:
     `except` suite ran to completion
 
 ```py
-def could_raise_returns_str() -> str:
-    return "foo"
+class A: ...
+class B: ...
+class C: ...
 
-def could_raise_returns_bytes() -> bytes:
-    return b"foo"
+def could_raise_returns_A() -> A:
+    return A()
```

Comment:
We could get rid of all of these `could_raise_returns_X` functions and just call `X()` directly (which could also raise). This would decrease the verbosity a lot. On the other hand, it makes the "could raise" aspect in the tests a little less clear. I'm okay with both variants.

Reply:
> On the other hand, it makes the "could raise" aspect in the tests a little less clear.

Yeah, I kind prefer them how they are, for this reason. But I do agree that the test setup is somewhat unfortunately verbose.